windows/RNFS-vnext/RNFSManager.cs

```diff
@@ -379,6 +379,21 @@ public async void copyFile(string filepath, string destPath, JSValue options, IR
             }
         }
 
+        [ReactMethod]
+        public async void copyToPictures(string filepath, IReactPromise<bool> promise)
+        {
+            try
+            {
+                var srcFile = await StorageFile.GetFileFromPathAsync(filepath);
```

Comment:
Need to use `StorageFile` to copy to Pictures dir as `KnowFolders.PicturesLibrary.Path` returns null.